pyproject.toml

```
@@ -1,3 +1,96 @@
+[tool.poetry]
+name = "epyqlib"
+version = "0.0.0"
+description = ""
+authors = ["Alex Anker <alex.anker@epcpower.com>"]
+
```

Add the license line.
```suggestion
license = "GPL-2.0-or-later"

```